README.md

```diff
@@ -2,180 +2,177 @@
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/be93b718-a6df-402a-b4a4-855ba186c97d/deploy-status)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys) [![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
 
-This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
-
-- [Contributing to the docs](#contributing-to-the-docs)
-- [Localization READMEs](#localization-readmes)
+Welcome! This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad you're interested in contributing.
+
+## Table of Contents
+1. [Using This Repository](#using-this-repository)
+2. [Prerequisites](#prerequisites)
```

Comment:
Please try avoid using explicit numbers in an ordered list. I.e., 

```suggestion
1. [Prerequisites](#prerequisites)
```

Actually, I don't think we want an ordered list here because the sections below may
be changed in the future. This order of section is not stable. So an unordered list
sounds better.

```suggestion
- [Prerequisites](#prerequisites)
```